packages/scss/src/components/_button.scss

```diff
@@ -62,6 +62,11 @@
 			background: _color($palette, "light");
 		}
 
+		&:focus {
+			box-shadow: inset 0 0 0 2px _color($palette, "200");
```

Comment:
Looking at the mock-up, it shouldn't be a inset box-shadow. The focus is around the button not inside. 
Also, the mock-up shows a 4px focus ring.